src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.GetCurrentThreadId.cs

```diff
@@ -1,15 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
corefx has a second copy of this file: https://github.com/dotnet/corefx/blob/master/src/Common/src/Interop/Windows/Kernel32/Interop.GetCurrentThreadId.cs

Reply:
Are you saying this code is dead and no one's using it?  Is corert not using it?

Reply:
- coreclr: dead code
- corefx: have their own copy
- corert: also have their own copy: https://github.com/dotnet/corert/blob/master/src/Common/src/Interop/Windows/Mincore/Interop.GetCurrentThreadId.cs

Reply:
Rather than deleting this one that's already being mirrored, wouldn't it be better to instead delete the other ones and share this one?

Reply:
I've sent PR against corefx but corert uses "api-ms-win-core-processthreads-l1-1-0.dll" instead of kernel32. AFAIR that was some kind of workaround related to Win7? Do you perhaps know if that is safe to change to kernel32?

Reply:
@krwq As long as the API documentation says "UWP apps" under supported platforms and "kernel32.dll" in DLL section (https://docs.microsoft.com/en-us/windows/desktop/api/processthreadsapi/nf-processthreadsapi-getcurrentthreadid) it is safe. It was already done for many other APIs.